hydra-node/test/Hydra/HeadLogicSpec.hs

```diff
@@ -600,6 +618,13 @@ observeTxAtSlot slot observedTx =
 observeTx :: OnChainTx SimpleTx -> Input SimpleTx
 observeTx = observeTxAtSlot 0
 
+observeConnectivityMessage :: TTL -> Connectivity -> Input SimpleTx
+observeConnectivityMessage ttl connectivityMessage =
```

Comment:
Note that `observe` in `observeTx` was only used to name it because the chain layer does "observe" transactions on chain. It is not wrong and does suit the network layer too, but any other verb would have also been fine in naming this. For example: `connectivityChanged :: Connectivity -> Input SimpleTx`